packages/terra-demographics-banner/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "terra-demographics-banner",
+  "main": "lib/DemographicsBanner.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "terra-demographics-banner",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-ui.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-demographics-banner",
+    "DemographicsBanner",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-ui/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-ui#readme",
+  "devDependencies": {
+    "terra-props-table": "^0.1.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
Lets do it just for this package now and we'll get another PR out to do it globally and get the generator updated.